app/src/main/java/org/p2p/wallet/newsend/ui/search/NewSearchPresenter.kt

```diff
@@ -184,4 +191,20 @@ class NewSearchPresenter(
         view?.clearUsers()
         view?.showNotFound()
     }
+
+    private fun logRecipientSelected(recipient: SearchResult) {
+        val address = when (recipient) {
```

Comment:
i suggest putting all this logic to analytics, it's not presentation logic and it will remove some lines from presenter